images/variables.tf

```
@@ -198,10 +198,15 @@ variable "third_party_bucket_name_prefix" {
 }
 
 variable "third_party_bucket_parameter_name" {
-  default     = "third_party_bucket_name"
-  description = "The name of the SSM Parameter Store parameter that will contain the name of the third-party S3 bucket."
+  default     = "/third_party_bucket_name"
+  description = "The name of the SSM Parameter Store parameter that will contain the name of the third-party S3 bucket, including the leading forward slash."
   nullable    = false
   type        = string
+
+  validation {
+    condition     = length(var.third_party_bucket_parameter_name) > 0 && substr(var.third_party_bucket_parameter_name, 0, 1) == "/"
```

I don't see any benefit to checking that the length is greater than zero- if someone passes in `""` as the value, it will still fail the "first character isn't a forward slash" check.

However, I do think we should check that the length is greater than 1 so that it rejects input of `"/"`.

This also raises the question of whether or not we should bother checking for any characters that are not allowed in SSM parameter names (per [here](https://docs.aws.amazon.com/systems-manager/latest/userguide/sysman-paramstore-su-create.html), the only valid characters other than slashes are `a-zA-Z0-9_.-`).
```suggestion
    condition     = length(var.third_party_bucket_parameter_name) > 1 && substr(var.third_party_bucket_parameter_name, 0, 1) == "/"
```

---

Please also see commit 796cc6f.